.github/workflows/documentation-windows.yml

```diff
@@ -0,0 +1,117 @@
+name: Documentation Windows
+
+on:
+  push:
+    branches: [ master, develop ]
```

Comment:
New workflow for documentation only on windows. will run on push to develop, which should be an acceptable frequency (not too often, but catch issues early)